src/main/kotlin/uk/gov/justice/digital/hmpps/supportadditionalneedsapi/resource/SearchController.kt

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.digital.hmpps.supportadditionalneedsapi.resource
+
+import jakarta.validation.constraints.Min
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.supportadditionalneedsapi.resource.model.SearchByPrisonResponse
+import uk.gov.justice.digital.hmpps.supportadditionalneedsapi.resource.model.SearchSortDirection
+import uk.gov.justice.digital.hmpps.supportadditionalneedsapi.resource.model.SearchSortField
+
+@RestController
```

Comment:
Skeleton controller class and method for the search API endpoint. This will be fleshed out in subsequent PRs